src/components/tools/edit/EditTool.js

```diff
@@ -0,0 +1,609 @@
+                component={FormTextField}
+            />
+            <Field
+                name="description"
+                label={t("toolDesc")}
+                id={build(parentId, ids.EDIT_TOOL_DLG.DESCRIPTION)}
+                component={FormMultilineTextField}
+            />
+            <Field
+                name="version"
+                label={t("toolVersion")}
+                id={build(parentId, ids.EDIT_TOOL_DLG.VERSION)}
+                required
+                validate={nonEmptyField}
+                component={FormTextField}
+            />
+            {isAdmin && (
+                <Field
+                    name="attribution"
+                    label={t("attribution")}
+                    id={build(parentId, ids.EDIT_TOOL_DLG.ATTRIBUTION)}
+                    component={FormTextField}
+                />
+            )}
+            {isAdmin && (
+                <Field
+                    name="location"
+                    label={t("location")}
+                    id={build(parentId, ids.EDIT_TOOL_DLG.LOCATION)}
+                    component={FormTextField}
+                />
+            )}
+            <Field name="type">
+                {({ field: { onChange, ...field }, ...props }) => (
+                    <FormSelectField
+                        {...props}
+                        label={t("type")}
+                        required
+                        field={field}
+                        onChange={(event) => {
+                            resetOnTypeChange(event.target.value, props.form);
+                            onChange(event);
+                        }}
+                        id={build(parentId, ids.EDIT_TOOL_DLG.TYPE)}
+                    >
+                        {toolTypes.map((type, index) => (
+                            <MenuItem
+                                key={index}
+                                value={type}
+                                id={build(
+                                    parentId,
+                                    ids.EDIT_TOOL_DLG.TYPE,
+                                    type
+                                )}
+                            >
+                                {type}
+                            </MenuItem>
+                        ))}
+                    </FormSelectField>
+                )}
+            </Field>
+            {isAdmin && (
+                <Field
+                    name="implementation"
+                    isAdmin={isAdmin}
+                    parentId={build(
+                        parentId,
+                        ids.EDIT_TOOL_DLG.TOOL_IMPLEMENTATION
+                    )}
+                    component={ToolImplementation}
+                />
+            )}
+            <Field
+                name={"container.image"}
+                parentId={parentId}
+                isOSGTool={isOSGTool}
+                component={ContainerImage}
+                t={t}
```

Comment:
Looks like `ContainerImage` is directly using the translation hook, so this prop can be removed.
```suggestion
```